monitoring/definitions/shared/container.go

```diff
@@ -87,3 +87,45 @@ var (
 		}
 	}
 )
+
+type ContainerMonitoringGroupOptions struct {
+	// CPUUsage transforms the default observable used to construct the CPU usage panel.
+	CPUUsage func(observable Observable) Observable
```

Comment:
I would make this a type since it's repeated so much:

```suggestion
	CPUUsage ObservableOption
```

Reply:
Updated.